docs/EXAMPLE_README_TEMPLATE.md

```diff
@@ -0,0 +1,120 @@
+# Arduino-ESP32 Example/Library Name ==(REQUIRED)==
+
+==*Add a brief description about this example/library here!*==
+
+This example/library demonstrates how to create a new example README file.
+
+# Supported Targets ==(REQUIRED)==
+
+==*Add the supported devices here!*==
+
+Currently, this example supports the following targets.
+
+| Supported Targets | ESP32 | ESP32-S2 | ESP32-C3 |
+| ----------------- | ----- | -------- | -------- |
+
+## How to Use Example/Library ==(OPTIONAL)==
+
+==*Add a brief description on how to use this example.*==
+
+* How to install the Arduino IDE: [Install Arduino IDE](https://github.com/espressif/arduino-esp32/tree/master/docs/arduino-ide).
+
+### Hardware Connection ==(OPTIONAL)==
+
+==*Add a brief description about wiring or any other hardware specific connection.*==
+
+To use this example, you need to connect the LED to the `GPIOx`.
+
+SDCard GPIO connection scheme:
+
+| SDCard Pin | Function | GPIO |
+| ----------- | -------- | ------ |
+| 1 | CS | GPIO5 |
+| 2 | DI/MOSI | GPIO23 |
+| 3 | VSS/GND | GND |
+| 4 | VDD/3V3 | 3V3 |
+| 5 | SCLK | GPIO18 |
+| 6 | VSS/GND | GND |
+| 7 | DO/MISO | GPIO19 |
+
+To add images, please create a folder `_asset` inside the example folder to add the relevant images.
+
+### Configure the Project ==(OPTIONAL)==
+
+==*Add a brief description about this example here!*==
+
+Set the LED GPIO by changing the `LED_BUILTIN` value in the function `pinMode(LED_BUILTIN, OUTPUT);`. By default, the GPIO is: `GPIOx`.
+
+#### Example for the GPIO4:
+
+==*Add some code explanation if relevant to the example.*==
+
+```cpp
+// the setup function runs once when you press reset or power the board
+void setup() {
+// initialize digital pin 4 as an output.
+pinMode(4, OUTPUT);
+}
+```
+
+#### Using Arduino IDE
+
+* Before Compile/Verify, select the correct board: `Tools -> Board`.
```

Comment:
What about adding here a link to Development Board overview? Users will be sure about main specification of their Board as Flash/PSRAM and also selected correct one if they are not sure.

[https://www.espressif.com/en/products/devkits]

Reply:
@pedrominatel are you going to adjust this?

Reply:
Done!